catroid/src/org/catrobat/catroid/io/StorageHandler.java

```diff
@@ -1120,4 +1120,9 @@ public static boolean deleteDirectory(File path) {
 		}
 		return path.delete();
 	}
+
```

Comment:
This should be called within XStreamToSupportCatrobatLanguageVersion0991AndBefore.java at line 584, so that it is called for all programs that contain the old version of the message, regardless if they are downloaded or locally opened.